astropy/units/format/console.py

```diff
@@ -40,10 +40,12 @@ def _format_unit_list(cls, units):
             if power == 1:
                 out.append(cls._get_unit_name(base_))
             else:
-                out.append('{}{}'.format(
-                    cls._get_unit_name(base_),
-                    cls._format_superscript(
-                            utils.format_power(power))))
+                out.append(
+                    '{}{}'.format(
+                        cls._get_unit_name(base_),
+                        cls._format_superscript(utils.format_power(power)),
+                    )
```

Comment:
```suggestion
                    str(cls._get_unit_name(base_)) + str(cls._format_superscript(utils.format_power(power)))
```

Reply:
Actually, both methods return `str` in the first place...